packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2887,6 +3007,45 @@ class IgnorePointer extends SingleChildRenderObjectWidget {
   }
 }
 
+/// Holds opaque meta data in the render tree.
```

Comment:
this could probably do with more motivation. It's not at all clear why you'd use this.